streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskExecutionMetadata.java

```diff
@@ -65,13 +67,17 @@ public boolean canProcessTask(final Task task, final long now) {
         final String topologyName = task.id().topologyName();
         if (!hasNamedTopologies) {
             // TODO implement error handling/backoff for non-named topologies (needs KIP)
+            log.trace("Task {} processing check for unnamed topology '{}'", task.id(), topologyName);
```

Comment:
I think we want to include this in the periodic logging, too? (same for the the new logs below?)